### PR TITLE
[WIP] attempt to fix OCI runtime failure

### PR DESCRIPTION
**Pull request by carolynhu**

### CODE REVIEW

File: perf/benchmark/runner/runner.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def kubectl_exec(pod, remote_cmd, runfn=run_command, container=None):
     c = ""
     if container is not None:
         c = "-c " + container
-    cmd = "kubectl --namespace {namespace} exec {pod} {c} -- {remote_cmd}".format(
+    cmd = "kubectl --namespace {namespace} exec {pod} {c} -- {remote_cmd} /bin/bash".format(
```
Comment (howardjohn) [inline perf/benchmark/runner/runner.py]: this doesn't seem right? can you add an explanation of the issue and why this fixes it

Comment (carolynhu) [reply perf/benchmark/runner/runner.py]: I made another PR for debugging purpose. I need to investigate more.

**Comment by carolynhu** [comment]
We have set debugging info into the perf pipeline, latest error log is:
```
kubectl --namespace twopods cp get_perfdata.sh fortioserver-7c6b79856f-gm4h7:/etc/istio/proxy/get_perfdata.sh -c istio-proxy
kubectl --namespace twopods exec fortioserver-7c6b79856f-gm4h7 -c istio-proxy -- ls /etc/istio/proxy/
envoy-rev0.json
kubectl --namespace twopods exec fortioserver-7c6b79856f-gm4h7 -c istio-proxy -- bash /etc/istio/proxy/get_perfdata.sh f1e37099_qps_1000_c_2_1024_mixer_srv_serveronly_perf.data 40
bash: /etc/istio/proxy/get_perfdata.sh: No such file or directory
command terminated with exit code 127
```

**Comment by carolynhu** [comment]
https://github.com/docker/for-linux/issues/246 `use /bin/sh instead of /bin/bash`

**Pull request closed without merging**